tests/migrations/test_runner.py

```diff
@@ -56,3 +57,29 @@ def test_version() -> None:
     assert runner._get_next_version(migration_key) == 2
     runner._update_migration_status(migration_key, Status.COMPLETED)
     assert runner._get_next_version(migration_key) == 3
+
+
+def test_no_schema_differences() -> None:
+    from snuba.datasets.schemas.tables import TableSchema
+    from snuba.datasets.storages import StorageKey
+    from snuba.migrations.parse_schema import get_local_schema
+
+    storages_to_test = [StorageKey.QUERYLOG]  # TODO: Eventually test all storages
+
+    runner = Runner()
+    runner.run_all()
+
+    for storage_key in storages_to_test:
+        storage = get_storage(storage_key)
+        conn = storage.get_cluster().get_query_connection(
+            ClickhouseClientSettings.MIGRATE
+        )
+
+        for schema in storage.get_schemas()._get_unique_schemas():
```

Comment:
please make get_unique_schemas fully public if you need to access it from outside the DatasetSchemas class and it is not just an implementation detail. `get_unique_schemas`.

Reply:
Updated